source/connect.txt

```diff
@@ -21,12 +21,11 @@ Prerequisites
 .. include:: /includes/fact-connect-prereqs.rst
    
 
-Local MongoDB Instance on Default Port
---------------------------------------
+Local Deployment on Default Port
```

Comment:
[suggestion] Each of these headings is really a mini-task. Would it make sense to title all of them with verb phrases, like tasks? 

For example:
- "Connect to a Local Deployment"
- "Connect to a Remote Deployment"
- "Connect with Options"